src/hacbs/components/PipelineRunDetailsView/PipelineRunDetailsSection.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import { SectionHeading } from '../../shared/utils/headings';
+import { PipelineRunKind } from '../../types';
+
+export interface PipelineRunDetailsSectionProps {
```

Comment:
Use `type` and do not export.
```suggestion
type PipelineRunDetailsSectionProps = {
```

Reply:
didn't wind up keeping this file